src/Sylius/Bundle/ProductBundle/Resources/config/serializer/Model.ProductAssociationType.yml

```diff
@@ -15,6 +15,18 @@ Sylius\Component\Product\Model\ProductAssociationType:
             expose: true
             type: array
             groups: [Default, Detailed]
-    virtual_properties:
-        getName:
-            serialized_name: name
+        createdAt:
```

Comment:
I just have checked a failing build. Becuase you have updated object serialization, response for product created with association should also be updated. We are missing `createdAt` and `updatedAt` there.